.travis.yml

```diff
@@ -19,36 +52,23 @@ env:
     # IPython 1.2.1 is the last version of IPython for python2.6
     # source: https://pypi.python.org/pypi/ipython/1.2.1
     - PYTHON=2.6 DEPS=full TEST=standard   # functionality
-    - PYTHON=2.7 DEPS=full TEST=standard IPYTHON_VERSION="==2"
-    - PYTHON=2.7 DEPS=minimal TEST=standard
+    - PYTHON=2.7 DEPS=full TEST=standard IPYTHON="=2" # NUMPY="=1.7"
     - PYTHON=3.4 DEPS=full TEST=standard
-
-    - PYTHON=2.7 DEPS=full TEST=examples  # test examples
-    - PYTHON=3.4 DEPS=full TEST=examples
-
-    - PYTHON=3.4 DEPS=minimal TEST=extra  # test file sizes, style, line endings
+    - PYTHON=3.4 DEPS=full TEST=examples  # test examples
+    - PYTHON=3.4 DEPS=minimal TEST=standard  # also tests file sizes, style, line endings
 
 
 before_install:
     - REDIRECT_TO=/dev/stdout  # change to /dev/null to silence Travis
```

Comment:
There was a span of time (a week or a month?) where the folding didn't work properly, so it was necessary back then. Now it's probably not necessary, but also isn't doing too much harm, so we'll probably remove it eventually.